public/partials/dashboards/edit.html

```diff
@@ -4,7 +4,9 @@
     <app-theme-link theme="selectedDashboard.theme"></app-theme-link>
 
     <div class="explorer-header" ng-init="initForm()">
-
+        <a href="/" class="btn-home">
```

Comment:
When I click this button, the page is refreshed. Try `href="home"` instead

Reply:
I changed it on every view